### PR TITLE
Modifytimestamp

### PR DESCRIPTION
**Pull request by sfi2022**
**Description**
Only add two attributes to attribute mapper. It helps me to simplify my implementation because i can use the mapper and dto infrastructure in ezLDAP. Otherwise i have to duplicate mapper and dto classes too. 

Short description or comments

**Reference**
https://github.com/it-at-m/zammad-ldap-sync

Issues #XXX

### CODE REVIEW

File: lib-spring/src/main/java/de/muenchen/oss/ezldap/spring/rest/v1/LdapUserController.java [hunk 1]
```diff
@@ -179,7 +179,8 @@ public ResponseEntity<List<LdapBaseUserDTO>> searchUser(
                     @ApiResponse(responseCode = "404", description = "OU existiert nicht"),
             }
     )
-    public ResponseEntity<List<LdapBaseUserDTO>> findByOu(@RequestParam(name = "ou") final String ou) {
+    public ResponseEntity<List<LdapBaseUserDTO>> findByOu(@RequestParam(name = "ou") final String ou,
+            @RequestParam(name = "modifyTimeStamp") String modifyTimeStamp) {
```
Comment (FabianWilms) [inline lib-spring/src/main/java/de/muenchen/oss/ezldap/spring/rest/v1/LdapUserController.java:183]: You add this parameter to this method, but it doesn't get used.

File: lib-core/src/test/java/de/muenchen/oss/ezldap/core/LdapServiceIntegrationTest.java [hunk 2]
```diff
@@ -22,11 +22,6 @@
  */
 package de.muenchen.oss.ezldap.core;
 
-import static org.assertj.core.api.Assertions.assertThat;
```
Comment (FabianWilms) [inline lib-core/src/test/java/de/muenchen/oss/ezldap/core/LdapServiceIntegrationTest.java:25]: These changes are not needed

File: lib-core/pom.xml [hunk 3]
```diff
@@ -50,6 +50,7 @@
 			<groupId>org.springframework.ldap</groupId>
 			<artifactId>spring-ldap-core</artifactId>
 		</dependency>
+
```
Comment (FabianWilms) [inline lib-core/pom.xml:53]: This change is not needed.

**Comment by eidottermihi** [comment]
@sfi2022 

>  It helps me to simplify my implementation because i can use the mapper and dto infrastructure in ezLDAP

But this just moves code that is only needed by you / zammad-ldap-sync to ezLDAP. Also I don't see a problem in "duplicating" mapper and dto infastructure in your codebase, maybe you can also remove a lot of stuff too which zammad does not need.

As we discussed in #12 I'm not in favour of including this in ezLDAP.

**Pull request closed without merging**